src/text/2-package-subsections/estimators.tex

```diff
@@ -1,19 +1,89 @@
 \subsection{gammapy.estimators}
 \label{ssec:gammapy-estimators}
 \todo{Axel Donath}
+By fitting parametric models to the data, the total integrated
+flux and overall temporal, spectral and morphological shape of the
+\gammaray emission can be constrained. In many cases it useful
+to make a more detailed follow-up analysis by measuring the
+flux in smaller spectral, temporal or spatial bins. This
+possibly reveals more detailed emission features, which
+are relevant for studying correlation with counterpart emissions.
+
 The \verb|gammapy.estimators| sub-module features methods to compute flux
 points, light curves, flux maps, flux profiles from data.
+The basic method for all these measurements is equivalent.
+The initial fine bins of \verb|MapDataset| are grouped into
+larger bins. The norm of the best fit "reference" spectral
+model is fitted ("scaled") in the  restricted data range only.
+
+The base class of all algorithms is the `Estimator`  class.
+
+Signifiance is estimated based on hypothesis \
+testing agains the background only model. This allows
+to assign a significance to the given flux bin.
+In addition to the best fit flux norm all estimators compute
+quantities corresponding to this flux. This includes
+the predicted number of total, signal and background
+counts per flux bin. The total fit statistics
+of the best fit model, the fit statistics of the
+null hypothesis and the difference between both.
+From the difference in test statistics the significance
+of the flux can be derived.
```

Comment:
significance of the flux, or the significance of the measured signal and the associated flux?